### ISSUE TITLE
chained function calls

### ISSUE THREAD

**Issue by du291** [spion/thenlint#4]
In the following case, no warning is raised, but it should qualify as the "missing return value" problem:

```
.then(function() { return boo(123); })
...
function boo(parameter) {
   q.ninvoke(... do something with parameter) // missing return here
}

```
